pipelines/ci-kubernetes-pipeline.yml

```diff
@@ -6,6 +6,19 @@ resources:
     uri: git@github.com:ONSdigital/census-rm-deploy.git
     private_key: ((github-private-key))
 
+- name: census-rm-kubernetes-repo
+  type: git
+  source:
+    uri: git@github.com:ONSdigital/census-rm-kubernetes.git
+    private_key: ((github-private-key))
+
+- name: census-rm-terraform
+  type: git
+  source:
+    branch: quiet-apply
```

Comment:
To be removed once https://github.com/ONSdigital/census-rm-terraform/pull/51 has been merged into `master`.